posts/Gunnar/2018-02-26-putting-bean-validation-constraints-to-multimaps.adoc

```diff
@@ -0,0 +1,217 @@
+= "Putting Bean Validation Constraints to Guava's Multimap"
+Gunnar Morling
+:awestruct-tags: [ "Discussions", "Hibernate Validator" ]
+:awestruct-layout: blog-post
+---
+
+The most significant feature in http://beanvalidation.org/news/2017/08/07/bean-validation-2-0-is-a-spec/[Bean Validation 2.0] (JSR 380) is the support for container element constraints.
+I.e. you can now apply constraints to the contents of container types such as `List`, `Map` or `Optional` by annotating their type arguments
+(which became possible with Java 8).
```

Comment:
Maybe a more graphic example could help? Not sure how people are used to annotate type arguments.